pkg/investigations/CannotRetrieveUpdatesSRE/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+name: cannotretrieveupdatessre
+rbac:
+  roles: []
+  clusterRoleRules:
+    - verbs:
+        - "get"
+        - "list"
+      apiGroups:
+        - "config.openshift.io"
+      resources:
+        - clusterversions
+customerDataAccess: false
```

Comment:
Should this be `true`?

Reply:
Nevermind, this relates to lockbox to prevent investigations against objects/data that a customer would consider sensitive